javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentTracerProviderConfigurer.java

```diff
@@ -38,6 +38,13 @@ private static SdkTracerProviderBuilder configure(
     if (Config.get().getBoolean(ADD_THREAD_DETAILS, true)) {
       sdkTracerProviderBuilder.addSpanProcessor(new AddThreadDetailsSpanProcessor());
     }
+    sdkTracerProviderBuilder.addSpanProcessor(new HeliosProcessor());
+
+    String heliosRatioProperty = getHeliosSamplingRationProperty();
```

Comment:
Please parse the double inside the getHeliosSamplingRationProperty method, so that you can catch failures